src/Data/MonadicStreamFunction/Core.hs

```diff
@@ -149,27 +131,9 @@ liftTransS = morphS lift
 -- | Apply trans-monadic actions (in an arbitrary way).
 --
 -- This is just a convenience function when you have a function to move across
--- monads, because the signature of 'morphGS' is a bit complex.
-morphS :: (Monad m2, Monad m1)
-      => (forall c . m1 c -> m2 c)
-      -> MSF m1 a b
-      -> MSF m2 a b
-morphS morph = morphGS morph'
-  where
```

Comment:
Since `morphS` is no longer uses the very generic `morphGS`, the explanation below is no longer unnecessary.